node/config/load.go

```diff
@@ -17,15 +18,40 @@ import (
 
 // FromFile loads config from a specified file overriding defaults specified in
 // the def parameter. If file does not exist or is empty defaults are assumed.
-func FromFile(path string, def interface{}) (interface{}, error) {
+func FromFile(path string, def interface{}, lc loadabilityCheck) (interface{}, error) {
```

Comment:
This isn't a "loadability" check, it's a config loading config. I'd:

1. Use the functional options pattern, or maybe a struct. Definitely not a private interface.
2. Call these config loading options, or something like that.